raiden/messages.py

```diff
@@ -853,18 +858,20 @@ def __init__(
         # guarantee that `amount` can be serialized using the available bytes
         # in the fixed length format
         if amount < 0:
-            raise ValueError('amount {} needs to be positive'.format(amount))
+            raise ValueError(f'amount {amount} needs to be positive')
 
         if amount >= 2 ** 256:
-            raise ValueError('amount {} is too large'.format(amount))
+            raise ValueError(f'amount {amount} is too large')
 
         if expiration < 0:
-            raise ValueError('expiration {} needs to be positive'.format(amount))
```

Comment:
Hmm I think there was an error here before and you copied pasted it. It should be:

```python
f'expiration {expiration} needs to be positive'
```